quic/transport/ngtcp2/stream/closedstate.nim

```diff
@@ -47,7 +47,7 @@ method leave*(state: ClosedStream) =
 method read*(state: ClosedStream): Future[seq[byte]] {.async.} =
   if not state.frameSorter.isComplete():
     let incomingFut = state.remaining.get()
-    if (await race(incomingFut, state.cancelRead)) == incomingFut:
+    if (await race(incomingFut, state.cancelRead, timeout(1.seconds))) == incomingFut:
```

Comment:
Hm, i'm not sure this timeout is the right solution, but instead quic timeout should likely be the one firing instead, as it is possible to be idle due to the other end taking time to reply,

Reply:
Ah! it's only being used in the handshake: https://github.com/vacp2p/nim-quic/blob/main/quic/connection.nim#L165

Reply:
Gonna try with this: https://github.com/vacp2p/nim-quic/pull/73

Reply:
Seems to work :thinking: will try running it few times